jobbergate-api/jobbergate_api/apps/job_scripts/services.py

```diff
@@ -62,7 +62,10 @@ async def auto_clean_unused_job_scripts(self) -> AutoCleanResponse:
         result = AutoCleanResponse(archived=set(), deleted=set())
 
         subquery = (
-            select(JobSubmission.job_script_id, func.max(JobSubmission.created_at).label("last_used"))  # type: ignore
+            select(
+                JobSubmission.job_script_id,
+                func.max(JobSubmission.created_at).label("last_used"),
+            )  # type: ignore
```

Comment:
Should this function set a limit since it's sorting by last used? That way, it only grabs N most recent jobs?

Reply:
Is it sorting? The goal here is to get the newest job-submission for each job-script, so we can archive/delete it

Reply:
Oh, I misread it. It's selecting the max. disregard.